src/controllers/buyProduct.controller.js

```diff
@@ -0,0 +1,57 @@
+import Cart from '../models/cart';
+import Product from '../models/Product';
+
+export class buyProduct {
+  static async getPdts(req, res, next) {
+    try {
+      const pdts = await Product.find({});
```

Comment:
Try to give your variables a name that is easily understandable like products instead of pdts.

Reply:
I'll refactor it to include this. Yes you can add multiple products to the cart.

Reply:
Since the last PR I've been getting this error `The engine "node" is incompatible with this module. Expected version "10.x". Got "12.14.0"` any idea why?

Reply:
I can't start the application, so I haven't tested the functionality but It should work.

Reply:
Please try to run it, cause on the add to cart url i cant see any url that is accepting parameters, then capture the id of the buyer.